cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -103,7 +103,15 @@ set(configure_command
     ${configure_command})
 
 # Pass C flags through
-set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
+string(REGEX MATCHALL "[^ ]+" PARSED_C_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
I think this logic should be applied not for all cases but only for problematic. Can you add a link to original issue? It was about OSX?

Reply:
I guess so. Let me limit it to OSX where `CMAKE_POSITION_INDEPENDENT_CODE` is `ON`.

Reply:
#1109 was the first time I saw this.